include/storage.h

```diff
@@ -42,6 +42,27 @@ inline void storage_store(void *storage, const uint8_t *key, const uint8_t *valu
     storage_store_bytes32(key, value);
 }
 
+/**
+ * transient_load / store load or store a value from transient storage accordingly.
+ * 
+ * The value of the first "storage" pointer is not used.
+ * generated headers provide:
+ *  a const storage pointer when working for a view-only function
+ *  a non const pointer for a mutating function
+ *  no pointer (so don't call transient_load) for a pure function
+ *
+ */
+inline void transient_load(const void* storage, const uint8_t *key, uint8_t *dest) {
+    transient_load_bytes32(key, dest);
+}
+
+/**
+ * see documentation for transient_load
+ */
+inline void transient_store(void *storage, const uint8_t *key, const uint8_t *value) {
+    transient_store_bytes32(key, value);
+}
+
```

Comment:
Let's drop these for now, since we're not yet supporting higher-order affordances

Reply:
:+1: